src/opentracing_agent.cpp

```diff
@@ -0,0 +1,18 @@
+#include <datadog/opentracing.h>
```

Comment:
Comment at the top about why we have two versions, and a comment on each about what they're for.

Reply:
Also remove this and re-add it using `git mv opentracing.cpp opentracing_agent.cpp`

Reply:
Comment added.